modules/network/pre-existing-subnetwork/metadata.yaml

```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 "Google LLC"
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+
+spec:
+  requirements:
+    services:
+    - compute.googleapis.com
```

Comment:
Please add 

```yaml
ghpc:
  has_to_be_used: true
```

This would prevent users from instantiating this module, but forgetting to use it. 
`pre-existing-vpc` should have it set as well, but we missed it (no need to address it as part of this PR).